apps/betterangels-backend/notes/models.py

```diff
@@ -196,7 +196,10 @@ class Note(BaseModel):
     _private_details: Optional[str]
 
     def __str__(self) -> str:
-        return self.purpose
+        if self.purpose:
```

Comment:
In code golf this can be

```python
def __str__(self) -> str:
    return self.purpose or str(self.id)
```

Reply:
yea, I got a mypy error for that but it looks like it was a false positive cus it's gone after restart